integreat_cms/cms/views/imprint/imprint_sbs_view.py

```diff
@@ -17,13 +20,17 @@
 
 @method_decorator(permission_required("cms.view_imprintpage"), name="dispatch")
 @method_decorator(permission_required("cms.change_imprintpage"), name="post")
-class ImprintSideBySideView(TemplateView):
+class ImprintSideBySideView(TemplateView, MediaContextMixin, ContentEditLockMixin):
     """
     View for the imprint side by side form
     """
 
     #: The template to render (see :class:`~django.views.generic.base.TemplateResponseMixin`)
     template_name = "imprint/imprint_sbs.html"
+
+    #: The url name of the view to show if the user decides to go back (see :class:`~integreat_cms.cms.views.mixins.ContentEditLockMixin`)
+    back_url_name = "edit_imprint"
+
```

Comment:
```suggestion
```
I think it would be better to redirect the user to the dashboard, because if someone else has taken over, the `edit_imprint` page also can't be edited.
Unfortunately the `ContentEditLockMixin` cannot handle redirecting to the dashboard, so it has to be done manually. Github does not let me add a suggestion, but you can add this:
```python
    def get_context_data(self, **kwargs):
        r"""
        Returns a dictionary representing the template context
        (see :meth:`~django.views.generic.base.ContextMixin.get_context_data`).

        :param \**kwargs: The given keyword arguments
        :type \**kwargs: dict

        :return: The template context
        :rtype: dict
        """
        context = super().get_context_data(**kwargs)
        context.update(
            {
                "back_url": reverse(
                    "dashboard",
                    kwargs={
                        "region_slug": kwargs["region_slug"],
                    },
                )
            }
        )
        return context

```

Reply:
May I ask you to share user steps to get redirected to the dashboard?
If I wanted to go back, I would use the button "Back to the imprint form" or "back" in the browser...
I can't get the other options :(